framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -199,6 +199,12 @@
     // org.checkerframework.framework.source.SourceeVisitor.checkForSuppressWarningsAnno
     "warnUnneededSuppressions",
 
+    // Require that warning suppression annotations contain a checker key as a prefix in order for
+    // the warning to be suppressed.
+    // org.checkerframework.framework.source.SourceChecker.checkSuppressWarnings(java.lang.String[],
```

Comment:
What is the purpose of these two lines of comment?  Is it a cut-and-paste error?

Reply:
The comment at the top of the `@SupportedOptions` annotation says:
```
    // When adding a new standard option:
    // 1. Add a brief blurb here about the use case
    //    and a pointer to one prominent use of the option.
```
This is the pointer to the use of the option.